Library/Homebrew/compat/md5.rb

```diff
@@ -1,26 +1,35 @@
 class Formula
   def self.md5(val)
+    # Don't throw an error at load time, just install time.
     stable.md5(val)
   end
 end
 
 class SoftwareSpec
   def md5(val)
+    odie <<-EOS.undent
+      Using MD5 has been dropped for security reasons.
+      Please switch to SHA-256.
+    EOS
```

Comment:
> This sounds a lot like "we should allow users to have a warning but install formulae using `md5`"

Could you please stick with the facts? Nobody stated anything to that end. Everybody agreed that we don't want to load or even install formulae that use those deprecated and security-related methods. @xu-cheng and I only kindly asked to use a solution that doesn't kill the Homebrew process, e.g., in the middle of a `brew tap`, leaving the user with a tap that contains such a broken formula, because `odie` intervened before `readall` could finish its work and `brew tap` could untap the problematic tap again.

If this doesn't make clear why `odie` is problematic then I don't know what else does and I guess I have to give up …